src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -610,6 +619,34 @@ public static DynamoModel Start(IStartConfiguration configuration)
         // Token representing the standard library directory
         internal static readonly string StandardLibraryToken = @"%StandardLibrary%";
 
+
+        private void InitializeLuceneConfig()
+        {
+            addedFields = new List<string>();
+
+            DirectoryInfo webBrowserUserDataFolder;
+            var userDataDir = new DirectoryInfo(pathManager.UserDataDirectory);
+            webBrowserUserDataFolder = userDataDir.Exists ? userDataDir : null;
+
+            string indexPath = Path.Combine(webBrowserUserDataFolder.FullName, "Index");
+            indexDir = Lucene.Net.Store.FSDirectory.Open(indexPath);
+
+            // Create an analyzer to process the text
+            SearchModel.Analyzer = new StandardAnalyzer(Configurations.LuceneNetVersion);
+
+            //Create an index writer
+            IndexWriterConfig indexConfig = new IndexWriterConfig(Configurations.LuceneNetVersion, SearchModel.Analyzer);
+            indexConfig.OpenMode = OpenMode.CREATE;
+
+            // When running parallel tests several are trying to write in the AppData folder then the job
+            // is failing and in a wrong state so we prevent to initialize Lucene when we are in test mode.
+            // TODO: make Lucene writer a singleton
+            if (!IsTestMode)
+            {
+                writer = new IndexWriter(indexDir, indexConfig);
```

Comment:
@reddyashish @RobertGlobant20 @zeusongit This line cant be run in parallel as unit tests